exalearn-ui/src/app/components/question/question.component.scss

```diff
@@ -0,0 +1,48 @@
+.radio-choice-group {
+	padding: 5px;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+}
+.questionText {
+	width: 100%;
+	padding: 50px 40px;
+	font-size: 2.5rem;
+	line-height: 2.5rem;
+}
+.question {
+	margin-top: 20px;
+}
+.radio-btn {
+	display: block;
+	width: 500px;
+}
+.btn-wrapper {
+	margin: 10px 0;
+}
+:host ::ng-deep .mat-radio-container {
+	height: 25px;
+	width: 25px;
+}
+:host ::ng-deep .mat-radio-label {
+	height: 70px;
+	padding-left: 15px;
+}
+:host ::ng-deep .mat-radio-button {
+	margin-left: -15px;
+}
+:host ::ng-deep .mat-radio-outer-circle {
+	height: 25px;
+	width: 25px;
+}
+:host ::ng-deep .mat-radio-inner-circle {
+	height: 25px;
+	width: 25px;
+}
+:host ::ng-deep .mat-radio-button .mat-radio-ripple {
```

Comment:
You can place .mat-radio-ripple selector inside existing :host ::ng-deep .mat-radio-button selector above (line 32)